docs/uml-class-diagrams/database/production/DatabaseCommManagerImplementation/DatabaseCommManagerImplementation.puml

```diff
@@ -2,8 +2,19 @@
 
 package edu.oakland.production.database {
     class DatabaseCommManagerImplementation {
+        - DatabasePersistentStorage databasePersistentStorage
+        - int rfidData
+        - LocationDataPoint locDataPoint
+        - Trackdata trackData
         __
+        + DatabaseCommManagerImplementation(int rfidData,
+         \tLocationDataPoint locDataPoint,
+         \tTrackdata trackData)
```

Comment:
Does this class need access to these private variables? If not, they should be removed.
```suggestion
        __
        + DatabaseCommManagerImplementation(DatabasePersistentStorage databasePersistentStorage)
```